buildspecs/linux37.yaml

```diff
@@ -52,5 +52,5 @@ phases:
       # Grab the version from the setup.py file using a grep "look-behind" https://stackoverflow.com/questions/30776265/how-to-grep-for-value-in-a-key-value-store-from-plain-text
       # If this is attempted on any branch other than master, it should fail because the role only has permission to push to the master branch
       # Colon (or maybe semi-colon) is causing issue with CodeBuild YAML parser, so needed to wrap whole command in single quote
-      - 'if [ "$PUBLISH_TO_PYPI" = "1" ] ; then if [ "$SHA_OF_LOCAL_COMMIT" = "$SHA_OF_REMOTE_MASTER" ]; then echo "Current branch matches SHA of remote master ($SHA_OF_REMOTE_MASTER) so publishing is allowed"; else echo "Current branch is not master. SHA of current commit is $SHA_OF_LOCAL_COMMIT but SHA of remote master is $SHA_OF_REMOTE_MASTER" && exit 1; fi && echo "Publishing to PyPI" && python3 setup.py bdist_wheel && git tag "$(grep -Po "(?<=    version=\")[0-9.]+" setup.py)" && echo "Tag applied, attempting to push" && git push --tags && echo "Changing wheel name to trick PyPI and then uploading" && cp ./dist/mantarray_waveform_analysis-0.3.1-cp37-cp37m-linux_x86_64.whl ./dist/mantarray_waveform_analysis-0.3.1-cp37-cp37m-manylinux1_x86_64.whl && rm ./dist/mantarray_waveform_analysis-0.3.1-cp37-cp37m-linux_x86_64.whl && twine upload dist/* ; fi'
+      - 'if [ "$PUBLISH_TO_PYPI" = "1" ] ; then if [ "$SHA_OF_LOCAL_COMMIT" = "$SHA_OF_REMOTE_MASTER" ]; then echo "Current branch matches SHA of remote master ($SHA_OF_REMOTE_MASTER) so publishing is allowed"; else echo "Current branch is not master. SHA of current commit is $SHA_OF_LOCAL_COMMIT but SHA of remote master is $SHA_OF_REMOTE_MASTER" && exit 1; fi && echo "Publishing to PyPI" && PACKAGE_VERSION=$(grep -Po "(?<=    version=\")[0-9.]+" setup.py) && python3 setup.py bdist_wheel && git tag "$PACKAGE_VERSION" && echo "Tag applied, attempting to push" && git push --tags && echo "Changing wheel name to trick PyPI and then uploading" && cp ./dist/mantarray_waveform_analysis-$PACKAGE_VERSION-cp37-cp37m-linux_x86_64.whl ./dist/mantarray_waveform_analysis-$PACKAGE_VERSION-cp37-cp37m-manylinux1_x86_64.whl && rm ./dist/mantarray_waveform_analysis-$PACKAGE_VERSION-cp37-cp37m-linux_x86_64.whl && twine upload dist/* ; fi'
```

Comment:
I forgot to mention that we should do this, but this will definitely make things easier when updating package versions